examples/permissioned-token/scripts/0-setup-token.js

```diff
@@ -0,0 +1,34 @@
+const { fundAccount, totalSupply } = require('./common/common');
+
+/**
+ * NOTE: this function is for demonstration purpose only (if ASA creator, manager are single accounts)
+ * If asset creator is a multisig address, then user should have a signed tx file, decoded tx fetched
```

Comment:
```suggestion
 * If asset creator is a multisig address, then user should have a valid tx file, add his 
```

Reply:
file will be signed already (by few of multisig accounts)

Reply:
It can be, it's not compulsory.